lib/screens/acknowledgement_sheet/acknowledgement_sheet.dart

```diff
@@ -11,6 +11,7 @@ import '../package_sheet.dart';
 part 'license_data.dart';
 
 class AcknowledgementSheet extends StatelessWidget {
+  static const String Acknowledgement = '/preference/acknowledgement';
```

Comment:
Can you name all the static variables same as 'route'. 

```diff
- static const String Acknowledgement = '/preference/acknowledgement';
+ static const String route = '/preference/acknowledgement';

Reply:
ok got it